src/components/OperatorList.tsx

```diff
@@ -78,13 +78,16 @@ const OperatorList: React.VFC<Props> = React.memo((props) => {
                   op.name
                 )}
               </span>
-              <span className="rarity" title={`Rarity: ${op.rarity + 1} stars`}>
-                <span className="rarity-number">{op.rarity + 1}</span>{" "}
```

Comment:
I think this space at the end of the line was for spacing purposes; are you sure it still looks right?